spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/zipkin/ZipkinConfigurations.java

```diff
@@ -56,6 +62,20 @@ Sender urlConnectionSender(ZipkinProperties properties) {
 					.build();
 		}
 
+		@Bean
```

Comment:
I have fixed a bug in `main` regarding the auto-configuration, I guess you'll have to rebase.